src/backend/Csrs.Api/Controllers/UserRequestController.cs

```diff
@@ -16,12 +16,15 @@ public UserRequestController(IMediator mediator, ILogger<UserRequestController>
 
         [HttpPost("Create")]
         [ProducesResponseType((int)HttpStatusCode.OK)]
-        public async Task<string> CreateAsync([Required] UserRequest request2)
+        public async Task<IActionResult> CreateAsync([Required] UserRequest userRequest, CancellationToken cancellationToken)
         {
-            Create.Request request = new();
-            Create.Response response = await _mediator.Send(request);
-
-            return string.Empty;
+            if (userRequest == null || userRequest.FileNo == null || userRequest.RequestType == null || userRequest.RequestMessage == null)
```

Comment:
You can also add `[Required]` on each of the properties and the framework will check this for you.  You can still leave this code here.